src/components/Datatable/Datatable.stories.tsx

```diff
@@ -150,6 +151,12 @@ export const Playground: Story<DatatableProps<Data>> = (args) => {
     tableConfig: argsTableConfig,
     ...restArgs
   } = args;
+
+  const onCancelLoading = () => {
```

Comment:
I'm just wondering why is call `onCancelLoading` and not just `onCancel`? Is there some special cancel loading?